src/auth/performance.service.ts

```
@@ -0,0 +1,81 @@
+@Injectable()
+export class PerformanceService {
+  constructor(
+    @InjectRepository(Application)
+    private readonly applicationRepo: Repository<Application>,
+    @InjectRepository(Job)
+    private readonly jobRepo: Repository<Job>,
+    @InjectRepository(User)
+    private readonly userRepo: Repository<User>,
+  ) {}
+
+   async getFreelancerPerformance(freelancerId: string): Promise<PerformanceMetricsDto> {
```

ESLint

[error] Replace `·async·getFreelancerPerformance(freelancerId:·string` with `async·getFreelancerPerformance(⏎····freelancerId:·string,⏎··`

(prettier/prettier)